src/MvvmNano.Forms/NanoMasterDetailPage.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Linq; 
+using Xamarin.Forms;
+
+namespace MvvmNano.Forms
+{
+    /// <summary>
+    /// A MasterDetailPage implementation that fits to the MvvmNano framework.
+    /// </summary>
+    public class NanoMasterDetailPage :  MasterDetailPage
```

Comment:
I am fine with the way you chose, but would like to have the naming consistent (MvvmNano instead of Nano) and both in one place (.cs).

```
class MvvmNanoMasterDetailPage
{
}

class NanoMasterDetailPage<T> : NanoMasterDetailPage
{   
}
```